tests/test_graph.py

```diff
@@ -24,6 +24,7 @@ def test_graphing(opgee_main, args):
         opgee_main.run(None, args)
         good = True
     except Exception as e:
+        import pdb; pdb.set_trace()
```

Comment:
This call to `pdb.set_trace()` should be deleted. BTW, this is pretty old school. Are you not using an IDE with a good debugger?

Reply:
`pdb` statement is gone now.

I do have a solid debugger with VSCode, but sometimes the amount of time required for it to startup is prohibitive, esp. in debugging tests.